api/main.py

```diff
@@ -112,5 +192,16 @@ def _healthcheck_ping():
 
 app.add_api_route(
     "/health",
-    health([_healthcheck_list_devices, _healthcheck_inference, _healthcheck_ping]),
+    health([_healthcheck_list_devices, _healthcheck_inference_e2e, _healthcheck_ping]),
+    summary="Check the health of the service",
+    description=(
+        "The healthcheck has checks more then whether the service is up. It will check"
```

Comment:
Eerste zin loopt niet echt. Bedoel je: "The healthcheck has checks in addition to whether the service is up."